src/Containers/ImageCarousel.tsx

```diff
@@ -11,7 +11,11 @@ export interface ImageCarouselProps
         Omit<React.HTMLAttributes<HTMLUListElement>, 'onClick'>,
         ImplicitPropsInterface {
     imageData: string[];
+    pointer:boolean;
     onClick: Function;
+    icon:boolean;
+    overlay:boolean;
+    overlayText:string;
```

Comment:
```suggestion
    icon: boolean;
    overlay: boolean;
    overlayText: string;
```

Reply:
I would have named it hoverOverlay to make it more clear :)

Reply:
or not sure what this does, is it a toggle for isHoverable? in this case the name hasOverlay etc might be better?